Cargo.toml

```diff
@@ -56,3 +56,7 @@ tower-http = { version = "0.6.6", features = [
   "compression-gzip",
   "decompression-gzip",
 ] }
+loom = { version = "0.7", optional = true }
+
+[features]
+loom = ["dep:loom"]
```

Comment:
Add `loom` to the dev dependency like: https://github.com/moka-rs/moka/blob/8dbc8e5c6ee804470b1144960f4065959658232a/Cargo.toml#L86-L87